packages/helm-charts/blockscout/values.yaml

```diff
@@ -131,6 +133,10 @@ blockscout:
             memory: 500Mi
             cpu: 700m
     pool_size: 10
+    sourcify:
```

Comment:
Isn't it overwritten by https://github.com/celo-org/celo-monorepo/pull/8476/files#diff-2d11e60d245b169b75ba864e9455b3204fb19624792b807100e1b8eaa20cc83fR154?

Reply:
I think that link is broken, it just redirects to this pull request changes page

Reply:
Ah, it's not broken, actually, it's just github's navigation. There is a `sourcify` section in this file on line 154. I think in yaml, it will take precedence.

Reply:
ah ok, I guess this is merge damage. I originally needed to add this to get helm to deploy.